packages.json

```diff
@@ -21850,5 +21850,19 @@
     "description": "MTProto client written in Nim",
     "license": "MIT",
     "web": "https://github.com/nimgram/nimgram"
+  },
+   {
+    "name": "jsontoxml",
+    "url": "https://github.com/MhedhebiIssam/jsontoxml/blob/main/jsontoxml.nim",
```

Comment:
That's not a correct link, you should link directly to the repository

Reply:
i had make new pull called json2ml that satisfy the requirements  , i just changed the name from jsontoxml to json2xml , check #1841